runtime/compiler/runtime/RelocationRuntime.cpp

```diff
@@ -1409,8 +1409,6 @@ TR_JITaaSRelocationRuntime::allocateSpaceInCodeCache(UDATA codeSize)
 
    uint8_t *coldCode;
    U_8 *codeStart = manager->allocateCodeMemory(codeSize, 0, &_codeCache, &coldCode, false);
-   // JITaaS FIXME: I think this code is needed too
```

Comment:
I don't yet know the implications of executing the code below. It seems to work without executing it.
Maybe leave exclude this FIXME until I have the time to think about it.